lib/ansible/modules/cloud/ovirt/ovirt_vm.py

```diff
@@ -522,21 +522,25 @@
         description:
             - "If I(true) C(kernel_params), C(initrd_path) and C(kernel_path) will persist in virtual machine configuration,
                if I(False) it will be used for run once."
+            - Not compatible with ovirt engine version 4.4+.
```

Comment:
I'd rephrase: `Parameter is usable with oVirt 4.3 and lower, but it's going to be removed in oVirt 4.4`